### PR TITLE
Update config.md

### PR DESCRIPTION
**Pull request by mellbratt**
Added a line about different ports used for other that the legacy port forwarder

### CODE REVIEW

File: _content/gateways/kerlink/config.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Password: pdmk-0b0e032d
 
 ### On the gateway
 
-Use the following command on the gateway to check whether data is being sent and received:
+Use the following command on the gateway to check whether data is being sent and received (change the port number depending on the packet forwarder used (e.g. 1901 for the new packet forwarder)):
```
Comment (johanstokking) [inline _content/gateways/kerlink/config.md:67]: Aren't these instructions using the Semtech UDP based packet forwarder?

**Comment by mellbratt** [comment]
That is possible. If I want to add instructions for "the new packet forwarder" should I do it on the same level as this doc in the wiki?

**Comment by johanstokking** [comment]
Yes. In fact we have instructions: https://github.com/TheThingsNetwork/packet_forwarder/blob/master/docs/INSTALL_INSTRUCTIONS/KERLINK.md

I think we need to stick with one official documented page for gateways. Currently it's the Semtech UDP packet forwarder based. We're currently working on a Gateway Agent, a new project that will replace our TTN Packet Forwarder. I think for the time being, let's keep the docs like this, skip the TTN Packet Forwarder and document the installation instructions that use the Gateway Agent.

**Comment by egourlao** [comment]
I'd also add that we have [packet forwarder-specific documentation sections](https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/legacy.html) in our documentation. There's no "troubleshooting" section yet for the Semtech packet forwarder, but it could be useful to have, given that multiple gateways can have common problems if they're running the Semtech packet forwarder.

**Comment by mellbratt** [comment]
Ok I just want to complement the available information so that others may get the Kerlink online more quickly. I can add to both those sections mentioned above instead.

**Pull request closed without merging**